### PR TITLE
EOS-23181: [server:hw:psu] Improve Description, add impact & recommendation.

### PR DESCRIPTION
**Pull request by sumedhak27**
Signed-off-by: Sumedh A. Kulkarni <sumedh.a.kulkarni@seagate.com>

## Coding:

* [x] Did you follow coding standard and verified it with flake8? 

* [x] Did your address all Codacy issues?

## Testing:

* [ ] Unittest updated and all unittests are passing?

* [x] Sanity/Self tests are updated (if applicable)?

* [ ] Manual testing done covering happy path and non-happy path?

## Integration:

* [ ] If there is any interface change, did you communicate it to other components?
* [ ] If there is any interface change, other componenent gate gatekeepers are ready to accept the change?
* [ ] Did you complete deployment test if applicable?

## PR checklist:
* [x] Did you add Jira number to the PR?

* [x] DCO and cla-signed?

### CODE REVIEW

**Review by shriya-deshmukh — APPROVED** [review]

**Review by sandeepsgt — APPROVED** [review]

File: low-level/sensors/impl/generic/node_hw_alerts_info.py [hunk 1]
```diff
@@ -47,5 +47,85 @@
...
+        **dict.fromkeys(["Presence detected", "Presence detected ()"], {
+            "Asserted": Alert(
+                "insertion", "informational",
+                "'Power Supply {}' is present. [{}]",
+                # Intel/Supermicro: 'Power Supply 1' is present. [PS1 (0x85)]
+                "Power Supply Port is being monitored.", "None"),
+            "Deasserted": Alert(
+                "missing", "critical",
+                "'Power Supply {}', is missing. [{}]",
+                "Power Supply Port monitoring has stopped.",
```
Comment (cdeshmukh) [inline low-level/sensors/impl/generic/node_hw_alerts_info.py:61]: Impact - Power supply redundancy is affected, if more power supplies also goes down or missing, server may go offline.
Recommendation - Install missing Power supply immediately

File: low-level/sensors/impl/generic/node_hw_alerts_info.py [hunk 2]
```diff
@@ -47,5 +47,85 @@
...
+                DEFAULT_RECOMMENDATION)
+        }),
+        **dict.fromkeys(["Predictive failure", "Predictive failure ()"], {
+            "Asserted": Alert(
+                "fault", "warning",
+                "Failure is predicted for 'Power Supply {}'. [{}]",
```
Comment (cdeshmukh) [inline low-level/sensors/impl/generic/node_hw_alerts_info.py:67]: Impact - Power supply redundancy will be affected, if more power supplies becomes faulty, server may go offline.
Recommendation - Replace faulty Power supply soon

change for other cases in similar fashion

File: low-level/sensors/impl/generic/node_hw_alerts_info.py (hunk 1 above)
Comment (sumedhak27) [reply low-level/sensors/impl/generic/node_hw_alerts_info.py]: changed. 👍

File: low-level/sensors/impl/generic/node_hw_alerts_info.py (hunk 2 above)
Comment (sumedhak27) [reply low-level/sensors/impl/generic/node_hw_alerts_info.py:68]: changed for other as well please review.

File: low-level/sensors/impl/generic/node_hw_alerts_info.py [hunk 3]
```diff
@@ -47,5 +47,91 @@
...
+            "Deasserted": Alert(
+                "fault_resolved", "informational",
+                "Configuration error resolved for 'Power Supply {}'. [{}]",
+                "Power Supply Redundancy is in good state.", "None"
+            )
+        }),
+        **dict.fromkeys(["Power Supply AC lost", "Power Supply AC lost ()"], {
+            "Asserted": Alert(
+                "fault", "critical",
+                "'Power Supply {}' lost the AC supply. [{}]",
```
Comment (cdeshmukh) [inline low-level/sensors/impl/generic/node_hw_alerts_info.py:110]: Recommendation -> Check power supply cable connectivity. If power cable also seen connected, cable may be faulty, check & replace it.

File: low-level/sensors/impl/generic/node_hw_alerts_info.py [hunk 4]
```diff
@@ -47,5 +47,91 @@
...
+            )
+        }),
+        **dict.fromkeys(["Power Supply Inactive",
+                         "Power Supply Inactive ()"], {
+            "Asserted": Alert(
+                "fault", "critical",
+                "'Power Supply {}' is inactive. [{}]",
+                "Power Supply Redundancy is affected. If more power " +
+                "supplies go down or missing, server may go offline.",
+                "Please Plug on the Power Supply Redundancy."
```
Comment (cdeshmukh) [inline low-level/sensors/impl/generic/node_hw_alerts_info.py:128]: Recommendation -> Try removing Power supply unit & inserting back, and check if recovered. If seen inactive still, PSU may need replacement.

**Review by cdeshmukh — APPROVED** [review]

**Pull request merged**